extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -129,14 +129,13 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
         .getVisibilityState();
     const adTestOn = this.element.getAttribute('data-adtest') ||
         isInManualExperiment(this.element);
-    let size;
     const width = this.element.getAttribute('width');
     const height = this.element.getAttribute('height');
-    if (width && height) {
-      size = {width, height};
-    } else {
-      size = this.getIntersectionElementLayoutBox();
-    }
+    // Need to ensure these are numbers since width can be set to 'auto'.
+    // Checking height just in case.
+    const size = (width && !isNaN(width) && height && !isNaN(height))
```

Comment:
isNaN works as you would expect on numeric strings, i.e., isNaN('250') == true. I will add the test as you asked.